lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
Based on @kohlhase 's report, https://github.com/KWARC/sTeX/issues/74, it is true that the spaces shouldn't be converted to nbsp in OpenMath.

If this is the case, what would be the best way to do the switch, so to differentiate between OpenMath and CMML?